iam-login-service/src/main/java/it/infn/mw/iam/notification/TransientNotificationFactory.java

```diff
@@ -245,8 +247,22 @@ public IamEmailNotification createGroupMembershipRejectedMessage(IamGroupRequest
   protected IamEmailNotification createMessage(String template, Map<String, Object> model,
       IamNotificationType messageType, String subject, List<String> receiverAddress) {
 
+    VelocityEngine ve;
```

Comment:
OMG, I didn't see this.

You are basically creating a new engine every time a message is sent, which is not what we want.

Reply:
Ah okay sorry I didn't realise that would be an issue. I did try and use the existing imported engine but it didn't work unfortunately.